v2/openapi/fb-provider-api.yaml

```diff
@@ -37,10 +37,12 @@ paths:
       summary: Describe server capabilities
       description: >-
         Returns the API version and all the capabilities that the server supports.  
-          
+        
         The capabilities are specified as a map. The map keys are the capability
         names and the values are lists of account IDs. If all the accounts support
-        a capability, an asterisk could be used, instead of listing all the accounts.
+        a capability, an asterisk could be used, instead of listing all the accounts. 
+        Exception is `crosstransfers` where it can accept only `*` or an empty list, 
+        it is not allowed to specify individual accounts.
```

Comment:
Generally speaking, I would like to avoid this kind of APIs. 
If there are some rules to the syntax, valid or invalid values - please describe it as an API spec, not as documentation. 
By the way, that itself is one of the reasons I don't like it—we changed the spec (mandating `accountId` as part of the path), but the document here states differently.